pontoon/base/static/js/fluent_interface.js

```diff
@@ -367,28 +390,29 @@ var Pontoon = (function (my) {
         }
       }
       else if (element.type === 'Placeable') {
-        if (element.expression.type === 'ExternalArgument') {
+        if (element.expression.type === 'VariableReference') {
           if (markPlaceables) {
             startMarker = '<mark class="placeable" title="External Argument">';
             endMarker = '</mark>';
           }
           string += startMarker + '{$' + element.expression.id.name + '}' + endMarker;
         }
-        else if (element.expression.type === 'MessageReference') {
+        else if (
+          element.expression.type === 'MessageReference' ||
+          element.expression.type === 'TermReference'
+        ) {
           if (markPlaceables) {
             startMarker = '<mark class="placeable" title="Message Reference">';
             endMarker = '</mark>';
           }
           string += startMarker + '{' + element.expression.id.name + '}' + endMarker;
         }
         else if (
-          [
-            'CallExpression',
-            'StringExpression',
-            'NumberExpression',
-            'VariantExpression',
-            'AttributeExpression',
-          ].indexOf(element.expression.type) >= 0
+          element.expression.type === 'AttributeExpression' ||
+          element.expression.type === 'CallExpression' ||
+          element.expression.type === 'VariantExpression' ||
+          element.expression.type === 'NumberLiteral' ||
+          element.expression.type === 'StringLiteral'
```

Comment:
Thanks for moving away from `indexOf`. This already looks easier on the eyes. I now realize that an even better way to handle those `ifs` would be a `switch` statement, with multiple cases falling through each other. It could be also used in `isSimpleElement`. I know you didn't want to spend too much time working on `fluent_interface`, so feel free to leave the code as-is for landing.